.landscaper/gardenlet/blueprint.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: landscaper.gardener.cloud/v1alpha1
+kind: Blueprint
+jsonSchema: "https://json-schema.org/draft/2019-09/schema" # required
+
+# imports are filled via the Installation CRD specifying an import section.
+imports:
+- name: runtimeCluster
+  targetType: landscaper.gardener.cloud/kubernetes-cluster
+- name: gardenCluster
+  targetType: landscaper.gardener.cloud/kubernetes-cluster
+- name: imageVectorOverwrite
+  schema:
+    type: string
+- name: componentImageVectorOverwrites
+  schema:
+    type: string
+- name: seedBackup
+  schema:
+    type: object
+    properties:
+      name:
+        type: string
+#       provider specific credentials just parsed as string and created as secret
+      providerCredentials:
+        type: object
+- name: revisionHistoryLimit
+  schema:
+    type: number
+- name: resources
+  schema:
+    type: object
+    properties:
+      requests:
+        type: object
+        properties:
+          cpu:
+            type: string
+          memory:
+            type: string
+      limits:
+        type: object
+        properties:
+          cpu:
+            type: string
+          memory:
+            type: string
+
+# check example/20-componentconfig-gardenlet.yaml for what can be configured
+- name: componentConfiguration
+  schema:
+    type: object
+
+deployExecutions:
+- name: default
+  type: GoTemplate
+  template: |
+    deployItems:
+    - name: deploy
+      type: landscaper.gardener.cloud/container
+      config:
+        apiVersion: container.deployer.landscaper.gardener.cloud/v1alpha1
+        kind: ProviderConfiguration
+
+        blueprint:
+          ref:
+            repositoryContext:
+    {{ toYaml ( index .cd.component.repositoryContexts (add (len .cd.component.repositoryContexts) -1) ) | indent 10 }}
```

Comment:
What's that? 👀

Reply:
@schrodit

Reply:
Basically it defines the current blueprint but we should make that more readable.
I will add the value as separate input value so that the blueprint can be referenced as `{{ toYaml .blueprint | indent 8 }}`